content/pages/healthcare/apply/application.md

```diff
@@ -3,9 +3,10 @@ title: Apply for Health Care
 entryname: hca
 layout: page-react.html
 body_class: page-healthcare
-in_maintenance: false
-maintenance_line1: We're sorry. The healthcare application is currently down while we fix a few things. We will be back up as soon as we can.
-maintenance_line2: In the meantime, you can still call 1-877-222-VETS(8387) and press 2 to complete this application over the phone.
+in_maintenance: true
+maintenance_line1: Scheduled Maintenance
```

Comment:
Jumping in here to say 1) let's stick with conversational language for error messages -- @akainic's suggestion works for me, and 2) I would keep the timing as specific as possible, but don't undershoot it. If there's a chance that it will take an extra 6 hours, let's buy ourselves some time.